pylate/indexes/voyager.py

```diff
@@ -144,6 +143,9 @@ def _create_collection(
             Whether to override the collection if it already exists.
 
         """
+        # We import voyager here because it is now an optional dependency
+        from voyager import Index, Space
```

Comment:
I think you can add like
```suggestion
        try:
            from voyager import Index, Space
        except ImportError:
            raise ImportError(
                "To use voyager index, please install it with `pip install pylate[voyager]`"
            )
```